.processes/bounty.md

```diff
@@ -0,0 +1,81 @@
+# Bounty Processes
+
+The HOPR Association _tech_ members, _community_ members, and [Project Owner](#legend) have agreed on the following bounty processes.
+The purpose of these processes is to follow a consistent way of creating, funding, and publishing bounties at HOPR.
+
+## The bounty
+
+A bounty usually serves multiple goals.
+
+1. Takes away workload from the _tech_ team as it allows external developers improve the HOPR protocol and the various applications/tools we maintain.
+2. Gives the _community_ team the opportunity to showcase HOPR's involvement in open source.
+3. Allows us to build a relationship between HOPR and the bounty hunter (tech community).
+
+The primary goal of a bounty creator is to take into account all 3 aspects, and depending on the bounty, adjust accordingly.
+
+## How to
+
+### Creating a bounty
```

Comment:
Who can create a bounty? Should this be specified? The previous section mentions "bounty creator" as a role, but it's not clear if there's a limit on who can initiate this.

Reply:
At this point in the company, multiple people have taken roles to create bounties, for now, I will not specify anything specific.